### PR TITLE
Navigation style

### PR DESCRIPTION
**Pull request by timhow38**
Ignore the Branch name it should be called "Visual Changes".
Changes that have been applied are as follows:

- Added a background image centered - Logo
- Created a Favicon and renamed the title of the site.
Note: Had changed some files to play around with the navigation however I have changed them back.

### CODE REVIEW

File: src/components/Header.js [hunk 1]
```diff
@@ -5,22 +5,23 @@ import { useContext } from 'react';
 import AppContext from './AppContext';
 import { Grid } from '@mui/material';
 
+
```
Comment (RileyHunter) [inline src/components/Header.js]: Space created

Comment (RileyHunter) [reply src/components/Header.js]:
```suggestion
```

File: src/components/Header.js [hunk 2]
```diff
@@ -5,22 +5,23 @@ import { useContext } from 'react';
...
-                    <Login />
-                </Grid>
-                <Grid item xs={4}>
-                    <CurrentElo />
-                </Grid>
-                <Grid item xs={4}>
-                    <RoutingButton text='Home' routeTarget='' />
-                </Grid>
-            </Grid>
+             context.user && <Grid container spacing={2}>
```
Comment (RileyHunter) [inline src/components/Header.js]: The old one was better IMO.

**Review by RileyHunter — APPROVED** [review]

File: src/components/Header.js (hunk 2 above)
Comment (timhow38) [reply src/components/Header.js]: Reverted

File: src/components/Header.js (hunk 1 above)
Comment (timhow38) [reply src/components/Header.js]: Removed Space

**Pull request merged**